docs/odata-rest-api.md

```diff
@@ -0,0 +1,836 @@
+
+In OData filter syntax the dot notation in the member name means a fully qualified type name. Walking down on the properties of the object tree is possible in the OData filter but the members' path must be separated by slash ('/') instead of dot ('.'). In this example you can see how to filter for an aspect field:
+
+```js
+$.ajax({
+    url: "/OData.svc/workspaces/articles?$filter=Summarizable/Summary eq 'sample value'",
+    dataType: "json",
+    async: false,
+    success: function (d) {
+        $.each(d.d.results, function () {
+            //...
+        });
+    }
+});
+```
+
+#### Auto filter and lifespan filter
+
+The following two parameters can be used to switch automatic filtering on or off (true or false).
+
+- enableautofilters
+- enablelifespanfilter
+
+Check the [autofilters](#AUTOFILTERS) and [lifespan](#LIFESPAN) filter articles for more information.
+
+Both setting's default value is false. The following query switches on both options:
+
+- http://example.com/odata.svc/workspaces/Project/budapestprojectworkspace?enableautofilters=true&enablelifespanfilter=true
+
+### $expand query option (supported from 6.2.1)
+
+According to the [OData protocol](http://www.odata.org/developers/protocols/uri-conventions#ExpandSystemQueryOption) the _$expand_ option indicates that related items should be represented inline in the response with full content instead of simple links. In our case this means that any [Reference Field](reference-field.md) can be expanded to be able to get metadata of a content and one or more related content with a **single HTTP request**.
+
+The value provided in the _$expand_ option is a **comma separated list of navigational properties** (in sensenet ECM these are reference fields). _$expand_ option works with a collection and a single content request as well. You may indicate that you want to expand **one or more fields** (e.g. `ModifiedBy` and `CreatedBy` at the same time). You may even expand fields of expanded content by providing a 'field name chain', separated by slashes (e.g. `CreatedBy/Manager`).
+
+Additionally, you may expand the following special fields as well:
+
+- **Actions**: list of available HTML actions
+- **AllowedChildTypes**: list of available content types in a container
+
+#### $expand and $select
+
+It is possible to specify the list of fields the response should contain (see $select option for more info). This works with expanded properties as well: you may specify wich fields of the expanded content should be added to the response by providing a 'field name chain', separated by slashes (e.g. _$select=CreatedBy/DisplayName_).
+
+If you do not provide a $select option in the request, all the field values will be returned of the requested and the expanded content as well. You do not have to select the property that you want to expand.
+
+#### Examples
+
+Expand the Manager user of all document workspaces:
+
+```js
+$.ajax({
+ url:/OData.svc/workspaces/document?$expand=Manager",	      
```

Comment:
There is a " missing I think. This might be what messes up the formatting everywhere later on:
```js
url:"/OData.svc/workspaces/document?$expand=Manager",
    dataType: "json",
 +    async: false,
 +    success: function (d) {
 +        $.each(d.d.results, function () {
 +		    //...
 +        });
 +    }
 +});
 +```